__tests__/index.test.js

```diff
@@ -243,4 +243,65 @@ describe('POST /api/reviews/:review_id/comments', () => {
                 expect(body.msg).toBe("Username not Found");
             });
     });
+});
+
+describe('PATCH : /api/reviews/:review_id', () => {
+    test('PATCH 200: updates review votes acording to passed object', () => {
+        return request(app)
+            .patch('/api/reviews/7')
+            .send({ inc_votes: 5 })
+            .expect(200)
+            .then(({ body }) => {
+                expect(body.updatedReviewVote[0]).toMatchObject({
+                    review_id: 7,
+                    title: 'Mollit elit qui incididunt veniam occaecat cupidatat',
+                    designer: 'Avery Wunzboogerz',
+                    owner: 'mallionaire',
+                    review_img_url:
+                        'https://images.pexels.com/photos/278888/pexels-photo-278888.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260',
+                    review_body:
+                        'Consectetur incididunt aliquip sunt officia. Magna ex nulla consectetur laboris incididunt ea non qui. Enim id eiusmod irure dolor ipsum in tempor consequat amet ullamco. Occaecat fugiat sint fugiat mollit consequat pariatur consequat non exercitation dolore. Labore occaecat in magna commodo anim enim eiusmod eu pariatur ad duis magna. Voluptate ad et dolore ullamco anim sunt do. Qui exercitation tempor in in minim ullamco fugiat ipsum. Duis irure voluptate cupidatat do id mollit veniam culpa. Velit deserunt exercitation amet laborum nostrud dolore in occaecat minim amet nostrud sunt in. Veniam ut aliqua incididunt commodo sint in anim duis id commodo voluptate sit quis.',
+                    category: 'social deduction',
+                    created_at: '2021-01-25T11:16:54.963Z',
+                    votes: 14
+                });
+            });
+
+    });
+    test('PATCH: 400 responds with error when object has wrong data', () => {
+        return request(app)
+            .patch('/api/reviews/7')
+            .send({ inc_votes: 'referendum' })
+            .expect(400)
+            .then(({ body }) => {
+                expect(body.msg).toBe('Bad Request');
+            });
+    });
+    test('PATCH: 400 responds with error when passed invalid review id', () => {
+        return request(app)
+            .patch('/api/reviews/garbage')
+            .send({ inc_votes: 5 })
+            .expect(400)
+            .then(({ body }) => {
+                expect(body.msg).toBe('Bad Request');
+            });
+    });
+    test('PATCH: 404 responds with error when passed valid but non existant review id', () => {
+        return request(app)
+            .patch('/api/reviews/777')
+            .send({ inc_votes: 5 })
+            .expect(404)
+            .then(({ body }) => {
+                expect(body.msg).toBe('Not Found');
+            });
+    });
+    test('PATCH: 404 responds with error inc_votes negative value turns votes into negative', () => {
```

Comment:
This shouldn't be a 404 error. Personally, I'd allow negative votes, but it depends how you want it to work on the front-end. Either way though, it's not a bad request. How about returning a vote set to 0 if the downvotes are more than the current total?